src/app/workflow/workflow.component.html

```diff
@@ -163,12 +163,13 @@ <h1 class="mat-display-2" *ngIf="currentTask.type === taskTypes.END_EVENT">{{wor
       </ng-container>
 
       <button
-        [disabled]="isLocked(currentTask)"
```

Comment:
We still need to check isLocked, and then **_also_** disable if this.loading is true.